python/cudf/cudf/_lib/labeling.pyx

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+import numpy as np
+from enum import IntEnum
+
+from libc.stdint cimport uint32_t
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+from cudf._lib.column cimport Column
+from cudf._lib.replace import replace_nulls
+
+from cudf._lib.cpp.labeling cimport inclusive
+from cudf._lib.cpp.labeling cimport label_bins as cpp_label_bins
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+
+
+# Note that the parameter input shadows a Python built-in in the local scope,
+# but I'm not too concerned about that since there's no use-case for actual
+# input in this context.
+def label_bins(Column input, Column left_edges, left_inclusive,
+               Column right_edges, right_inclusive):
```

Comment:
Can `left_inclusive` and `right_inclusive` be typed as `bool`?

Reply:
Sure.

Reply:
@shwina just to confirm, there's no way to type these arguments without doing something like the following, is there?
```
from libcpp import bool as cbool
def label_bins(..., cbool left_inclusive, ...):
```
I don't even know if that is actually restrictive w.r.t. function arguments, I've only ever found it useful when interacting with a C++ API that expects a bool (e.g. a C++ function that requires a bool must be typed in Cython using the above import).

Reply:
I think you can just mark it as `bool` without importing anything? I may be mistaken though.

Reply:
I'm pretty sure you'll run into issues like [this one](https://stackoverflow.com/questions/24659723/cython-issue-bool-is-not-a-type-identifier). Even a trivial test fails for me:
![image](https://user-images.githubusercontent.com/1538165/112551434-39336f80-8d7e-11eb-8415-749a642ee1a8.png)

Reply:
Yup, I was mistaken! I think in other files we typically do `from libcpp cimport bool` and use it.

Reply:
`libcpp.bool` sounds fine to me. I think we use in a bunch of places

Reply:
Done.